UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -55,20 +56,38 @@ public class Tab : INotifyPropertyChanged
         public static readonly BitmapImage ClosedIcon = new(new Uri(@"/Resources/X.png", UriKind.RelativeOrAbsolute));
         public static readonly BitmapImage ClosedHoverIcon = new(new Uri(@"/Resources/X_Down.png", UriKind.RelativeOrAbsolute));
 
-        // it's actually used, but when it will be compiled (see the note in "MainWindow" class)
-        #pragma warning disable CS0067
+        private static readonly MainWindow mainWindow = Application.Current.MainWindow as MainWindow;
```

Comment:
Why is a reference to itself necessary?
Can't you just use `this`?

Reply:
`Tab` class is separate from `MainWindow`, so `this` will return the `Tab` instance.

Reply:
Ah, missed that context from the web UI. Really need a way to get PR diffs into an IDE...